indico/web/client/js/react/hooks.js

```diff
@@ -21,15 +21,26 @@ export const useFavoriteUsers = () => {
   // once on the same page we keep the favorites in sync and don't send multiple
   // requests to load the initial list
   const [favorites, setFavorites] = useState({});
+  const [loading, setLoading] = useState(false);
+  const userId = null;
 
   const apiCall = async (method, id = null, source = null) => {
+    setLoading(true);
```

Comment:
Oh, I guess that's what caused the behavior we were talking about before! Maybe you could only set loading during the initial load of the favorite list, but not when performing a modification?